pkg/tests/pkg_deb_test.py

```diff
@@ -160,6 +160,7 @@ def test_description(self):
 	'Built-Using: some_test_data',
 	'Replaces: oldpkg',
 	'Breaks: oldbrokenpkg',
+    'Provides: hello',
```

Comment:
The indent does not look right here. Can you check?

Reply:
Hmm, there seems to be a mixture of tabs and spaces in the original file. I changed the indentation for this line to be a tab to match the existing style.